fe/fe-core/src/main/java/com/starrocks/persist/EditLog.java

```diff
@@ -800,7 +800,8 @@ public static void loadJournal(GlobalStateMgr globalStateMgr, JournalEntity jour
                 case OperationType.OP_MODIFY_BINLOG_AVAILABLE_VERSION:
                 case OperationType.OP_MODIFY_BINLOG_CONFIG:
                 case OperationType.OP_MODIFY_ENABLE_PERSISTENT_INDEX:
-                case OperationType.OP_ALTER_TABLE_PROPERTIES: {
+                case OperationType.OP_ALTER_TABLE_PROPERTIES:
+                case OperationType.OP_MODIFY_TABLE_CONSTRAINT_PROPERTY: {
                     ModifyTablePropertyOperationLog modifyTablePropertyOperationLog =
                             (ModifyTablePropertyOperationLog) journal.getData();
                     globalStateMgr.replayModifyTableProperty(opCode, modifyTablePropertyOperationLog);
```

Comment:
It Looks  like miss replay OP_MODIFY_TABLE_CONSTRAINT_PROPERTY at replayModifyTableProperty function

Reply:
no, I have modify it in buildProperty()